index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Country Documentation</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <main id="main-doc">
+    <section class="main-section" id="The_World">
+      <header>
+        <h1>The World</h1>
+      </header>
+      <article>
+        <p>
+          Hello, We are Abdol Hakem and Shihab Uddin Khan. We are here to talk about different things of countries.
+        </p>
+
+        <ul>
+          <li>Egypt</li>
+          <li>Bangladesh</li>
+          <li>USA</li>
+          <li>Canada</li>
+          <li>Dubai</li>
+        </ul>
+      </article>
+    </section><br>
+
+    <section class="main-section" id="Country_Egypt">
+      <header>
+        <h1>Country Egypt</h1>
```

Comment:
> * [ ] Kindly use  just a single `<h1>` tag throughout an HTML file which represents the main title of the website. Other headings should be `h2 - h6` which is the order of the hierarchy of headings for the  semantic reasons .

Reply:
- [ ] [OPTIONAL] Kindly consider referring the students to the reading materials that you think may explain more about your suggestion. For example, you may consider attaching the link that explain more about Semantic HTML so that the student can have a clear understanding ☺️.